src/components/eachBook.js

```diff
@@ -1,31 +1,74 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+// import './styles/Book.css';
```

Comment:
OPTIONAL
_I make it optional but this is a general requirement for every project._
- [ ] To make clean code kindly delete commented code.